yolov5/utils/plots.py

```diff
@@ -261,6 +294,21 @@ def plot_images(images, targets, paths=None, fname='images.jpg', names=None, con
         im = im.transpose(1, 2, 0)
         mosaic[y:y + h, x:x + w, :] = im
 
+    # There is a bug in YOLOv5 plots where the image colors are sometimes incorrectly inverted based on the image size,
```

Comment:
Also found this on git issues: https://github.com/ultralytics/yolov5/issues/11726